### DISCUSSION TITLE
Station G2 TX Power - before or after PA gain?

### DISCUSSION THREAD

**Discussion by mmiller7** [meshtastic/Discussions#150] [Firmware]
For the Station G2, is the PA gain factored into the "TX Power" field of the Station G2 Meshtastic firmware build?

In other words, to stay under 1W out at the antenna connector, would the firmware be set to 30dBm or 10dBm?

Per the chart on the Station G2 page, under "[Summary for Lora Power Amplifier Conduction Test](https://wiki.uniteng.com/en/meshtastic/station-g2#summary-for-lora-power-amplifier-conduction-test)"
 it looks like the LoRA module should output 10dBm to get 30dBm out of the PA circuit.  But I don't know which I should input to the software.

**Comment by garthvh** [comment]
10
